x-pack/plugins/observability/server/services/slo/historical_summary_client.ts

```diff
@@ -227,7 +232,7 @@ function generateSearchQuery(slo: SLO, dateRange: DateRange): MsearchMultisearch
       daily: {
         date_histogram: {
           field: '@timestamp',
-          fixed_interval: '1d',
+          fixed_interval: fixedInterval,
```

Comment:
💬 between the start and end date, we split the duration per fixed interval buckets (1hour to 1day, depending on the time window duration).